### DISCUSSION TITLE
Help with quasar v2 vuex module not recognizing

### DISCUSSION THREAD

**Discussion by phgrund** [quasarframework/quasar#9613] [General - Components / Directives / etc] [answered]
I'm not sure if it's the right category to post this, but I'm having trouble related to the vuex, not sure if its due to the typescript

![image](https://user-images.githubusercontent.com/25393880/121250945-b26e3900-c87c-11eb-9f7c-2ae41e17be25.png)

What I tried so far

- `quasar c && rm -rf node_modules && rm package-lock.json && npm i`
- `npm i vuex` (completely broke my project)

Any help?

Quasar versions when I use `quasar upgrade`
quasar: 2.0.0-rc.1 → undefined
@quasar/app: 3.0.0-rc.1 → undefined

**Comment by metalsadman** [comment, marked as answer]
https://github.com/quasarframework/quasar/releases

**Reply by rstoenescu** [reply]
More specifically: https://github.com/quasarframework/quasar/releases/tag/%40quasar%2Fapp-v3.0.0-rc.1
Please read the release notes on each upgrade, especially before getting out of beta and rc releases.
